pkg/ucp/types/ucpTypes.go

```diff
@@ -99,3 +99,21 @@ type InterlockConfig struct {
 	Arch             string `json:"Arch"`
 	InterlockEnabled bool   `json:"InterlockEnabled"`
 }
+
+// ClientBundles - configuration for Interlock layer 7 routing
```

Comment:
`configuration for Interlock layer 7 routing` - heh, guess thats a copy and paste error

Reply:
Haha ye I noticed a couple others as well, will tidy them up.